pkg/cloud/google/machineactuator.go

```diff
@@ -255,93 +255,109 @@ func (gce *GCEClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.Mach
 		return gce.handleMachineError(machine, verr, createEventAction)
 	}
```

Comment:
Can you also move the machineSetupConfigGetter checks to create? If an instanceTemplate is provided we don't need to rely on machineSetupConfigs.